modules/ROOT/pages/authentication-authorization/manage-roles.adoc

```diff
@@ -576,6 +582,13 @@ This is equivalent to running `DROP ROLE myrole IF EXISTS` followed by `CREATE R
 * The `CREATE OR REPLACE ROLE` command does not allow you to use the `IF NOT EXISTS`.
 ====
 
+[[access-control-immutable-roles]]
+=== Immutable roles
+Immutable roles are roles which cannot be modified in the usual way.
+This means they cannot be created, renamed, dropped, or have privileges granted to or revoked from them.
```

Comment:
This contradicts the above commands, for example, `CREATE [IMMUTABLE] ROLE name AS COPY OF otherName` or `CREATE OR REPLACE [IMMUTABLE] ROLE name [AS COPY OF otherName]`, etc.